app/src/main/java/com/onboarding/pronosticAR/mvp/view/base/FragmentView.kt

```diff
@@ -0,0 +1,15 @@
+package com.onboarding.pronosticAR.mvp.view.base
+
+import android.content.Context
+import androidx.fragment.app.Fragment
+import java.lang.ref.WeakReference
+
+open class FragmentView(fragment: Fragment) {
+    private val fragmentRef: WeakReference<Fragment> = WeakReference<Fragment>(fragment)
+
+    val fragment: Fragment?
+        get() = fragmentRef.get()
+
+    val context: Context?
+        get() = fragment?.context
+}
```

Comment:
mmm you not need a fragment now, remove this

Reply:
removed